src/css/my-styles.css

```diff
@@ -0,0 +1,8 @@
+.title {
+  text-align: center;
+  color: blue;
+}
+
+html {
+  margin: 30px;
+}
```

Comment:
Since you're using bootstrap, try to use it's styles unless they're missing in the library